tests/serverpod_test_server/test/generated_methods/to_json_test.dart

```diff
@@ -621,7 +621,7 @@ void main() {
       ),
     );
 
-    var jsonMap = object.toJson();
+    var jsonMap = object.toConstraintJson();
```

Comment:
Suggestion: The tests in this file should probably be moved to the `all_to_json_test.dart` file and then this file should contain tests that validate the behavior of the `toConstraintJson()` method.

Reply:
Please make sure to validate all test cases are needed and are adapted to the new names.

Reply:
We probably want a couple of more `toContraintJson()` scenarios as well that validate the different cases we can have. For example a List containing a class with server only fields.